### ISSUE TITLE
Since last version, JP no longer works at all

### ISSUE THREAD

**Issue by observeroftime02** [29988122/Fate-Grand-Order_Lua#341]
I updated to the latest release, now when running the script it just draws red squares on the screen in three different locations. Doesn't select anything, doesn't do anything. Had to revert to an older release. Here's some screenshots of it drawing the squares.

![image](https://user-images.githubusercontent.com/43228249/75331535-bddc5300-5837-11ea-8431-8afd1b0c7b98.png)

![image](https://user-images.githubusercontent.com/43228249/75331839-3f33e580-5838-11ea-978d-ce4e4a03ef55.png)

![image](https://user-images.githubusercontent.com/43228249/75331587-d187b980-5837-11ea-9f29-6be373fe0c9f.png)

The EN version of the script never ran on my Note 10+, but the JP version worked fine for a long time, until the latest major update of the script I guess. Now both the EN and JP version no longer work on either Note 10+ or Note 5. 

Here's a screenshot from the support screen on my Note 5, also just drawing the squares but not doing anything at all
![image](https://user-images.githubusercontent.com/43228249/75332106-ad78a800-5838-11ea-8fe3-72042d3be07f.png)

Furthermore, choosing "first" as a support option throws null pointer exception like here
https://github.com/29988122/Fate-Grand-Order_Lua/issues/340

**Comment by zxcvnnvcxz** [comment]
Do you have this line in your  `FGO_EN_REGULAR.lua` or `FGO_JP_REGULAR.lua`?
`debug = false -- set to 'true' if needing to debug`

**Comment by pozeromon** [comment]
> Do you have this line in your `FGO_EN_REGULAR.lua` or `FGO_JP_REGULAR.lua`?
> `debug = false -- set to 'true' if needing to debug`

yes I have that on fgo_en_regular.lua

**Comment by popkk213** [comment]
I have same problems too
![Screenshot_2020-03-01-16-51-48-94_13e7a6b1087387188a64bd1592e629e3](https://user-images.githubusercontent.com/49636684/75623546-8d047100-5bdd-11ea-93ad-4eb8564bd51e.png)
![Screenshot_2020-03-01-16-52-04-76_13e7a6b1087387188a64bd1592e629e3](https://user-images.githubusercontent.com/49636684/75623553-97bf0600-5bdd-11ea-8fc7-7586d04064b8.png)
![Screenshot_2020-03-01-16-51-55-03_13e7a6b1087387188a64bd1592e629e3](https://user-images.githubusercontent.com/49636684/75623561-b32a1100-5bdd-11ea-9567-fe2ddf3fb6c3.png)

**Comment by observeroftime02** [comment]
Changing the debug setting or removing it entirely doesn't change anything at all. The script simply draws the red squares and doesn't do anything further. Anyone got any ideas?

**Comment by zxcvnnvcxz** [comment]
For now just go back to the old release [https://github.com/29988122/Fate-Grand-Order_Lua/tree/0d8aba09170391b58b9ce740644e96c5b80636a5](url). There isn't much you are missing out on, unless you really want the retry function.

I only tested on nox emulator, which isn't the greatest platform to test on, but seems like being on accessibility on ankulua produces red squares while daemon doesn't. Don't know if info is relevant, but hopefully there are fixes in the future.

**Comment by reconman** [comment]
I found the issue, https://github.com/29988122/Fate-Grand-Order_Lua/blob/83226ac1feb894869375a0a93fb64ca4e0184ce7/regular.lua#L327 checks for a `debug` variable but debug is the name of the [Lua debug library](http://lua-users.org/wiki/DebugLibraryTutorial).

`print(debug)` will output something like "table: 0x5620b316dab0" and checking debug in an if or while condition will result in true, because the variable is neither nil nor false. The debug variable is only set in FGO_EN_REGULAR, not in FGO_JP_REGULAR.

I have fixed the issue (95 % probability) in my Pull Request.
